extensions/wikia/CreateNewWiki/tasks/CreateTables.php

```diff
@@ -55,21 +55,49 @@ public function run() {
 		$dbw = wfGetDB( DB_MASTER, [ ], $this->taskContext->getDBname() );
 		$this->taskContext->setWikiDBW( $dbw );
 
-		foreach ( $this->sqlFiles as $file ) {
-			$this->debug( __METHOD__ . ": Populating database with {$file}" );
+		$dbw->multiQuery( $this->getSchemaSQL(), __METHOD__ );
+		$dbw->commit( __METHOD__ );
```

Comment:
mysqli::multi_query is 16x faster than using DatabaseBase::sourceStream that sends query for each statement